lib/ClimaLandSimulations/src/Fluxnet/fluxnet_simulation.jl

```diff
@@ -0,0 +1,411 @@
+export fluxnet_simulation, fluxnet_simulation_test
+
+function fluxnet_simulation_test(

```

Comment:
what is `fluxnet_simulation` vs `fluxnet_simulation_test`?

Reply:
I removed fluxnet_simulation_test, it was just to test things locally